README.md

```diff
@@ -1,2 +1,272 @@
 # TIDES
-Transit ITS Data Exchange Specification for historical ITS data
+Transit ITS Data Exchange Specification (TIDES) for historical ITS data.
+
+This repository references TIDES as described in the [prepublication report](https://nap.nationalacademies.org/catalog/26674/improving-access-and-management-of-public-transit-its-data).
+
+
+## Files in Data Structure
```

Comment:
Readme should have "at a glance" topics, not the whole spec :-)
Suggest stubbing out things like:

- Installation (if/when applicable)
- General Usage
- Overview/organization
- Contributing / issues
- Governance
- Credits

Reply:
(feel free to suggest others /replace!  this is just OTTOMH)

Reply:
It may be good to add a Background section that includes a) info on the TIDES Project (and the brief genesis of the concept) and b) the linkage to the TCRP research report and the NASEM/TRB disclaimer.

Reply:
Definitely -- I'm not going to have the context necessary to generate any of this content. I'm happy to replace with an outline and stage copy as it becomes available though.

Reply:
Here is language for the Acknowledgement:

**TIDES**

Transit ITS Data Exchange Specification (TIDES) for historical transit operations data.

This repository provides data schemas and tools to support the access, management, and improvement of historical transit operations data, including vehicle operations, passenger activity, fare collection, and other similar data.

**Acknowledgment** 

These data schemas and tool definitions developed here are based on the results of research conducted by the Transportation Research Board (TRB) of the National Academies of Science, Engineering, and Medicine (NASEM) under the Transit Cooperative Research Program (TCRP).  This research is available at https://nap.nationalacademies.org/catalog/26674/improving-access-and-management-of-public-transit-its-data. 

Note that neither the TIDES Project nor the TIDES-transit repository are associated with TCRP, TRB, or the Academies.  The use of the TCRP research results in this repository do not reflect any explicit or implicit endorsement of or participation in this work by the Academies.  In addition, the opinions and conclusions expressed or implied in the research are those of the contractor. They are not necessarily those of the Transportation Research Board, the Academies, or the program sponsors.